storage/storage.go

```diff
@@ -16,6 +16,16 @@ import (
 	"strings"
 )
 
+// Operation is a type used to define funnel storage operations
+type Operation int
+
+const (
+	// Get represents the operation Get(ctx context.Context, rawurl string, hostPath string, class tes.FileType) error
+	Get Operation = iota
+	// Put represents the operation Put(ctx context.Context, rawurl string, hostPath string, class tes.FileType) error
+	Put
```

Comment:
Probably better if these are private

Reply:
Oh, except, I guess the Backend interface needs to be private too, then.

Reply:
The worker accesses these in the input and output validation steps. 

The other option is to have a two methods: `SupportsGet` and `SupportsPut`.